x/wasm/types/params.go

```diff
@@ -171,6 +208,13 @@ func (a AccessConfig) Allowed(actor sdk.AccAddress) bool {
 		return true
 	case AccessTypeOnlyAddress:
 		return a.Address == actor.String()
+	case AccessTypeAnyOfAddresses:
+		for _, v := range a.Addresses {
```

Comment:
Looks good logic.
"Not covered by tests" comments means this is not used?
I guess this is why it is draft